code/datums/uplink/ammunition.dm

```diff
@@ -83,7 +83,7 @@
 
 /datum/uplink_item/item/ammo/c545/ap/ext
```

Comment:
nitpick: confusing to have the `ap/ext` when it's `ext/ap` now, but doubt it will actually matter.